### ISSUE TITLE
When will the latest versions be pushed to MS Store?

### ISSUE THREAD

**Issue by scrutiny6239** [soleon/Percentage#11]
Okay. So the _settings not persisting_ issue is due to the version number. I have 2.0.6 but apparently it is fixed in 2.0.7 and 2.0.8. However, MS Store only provides versions up to 2.0.6. When should I expect MS Store to automatically update to 2.0.8?
Additionally, I would like to share that I do not have the folder 
`C:\Users\<user name>\AppData\Local\Percentage.App\`
in my computer under [that path](https://github.com/soleon/Percentage/issues/4#issuecomment-2543127490) (I only have one User so there is no mixup).
Thanks. Great app. However, I do not enjoy it popping up on startup every day.

**Comment by soleon** [comment]
@scrutiny6239 MS Store release can be delayed depending on how fast MS can finish the certification process of each version. I submitted 2.0.8 to the store the same time I released it on GitHub, however, 2.0.8 is still under certification at the time of writing.

If you don't have `C:\Users\<user name>\AppData\Local\Percentage.App\` I suspect that either your local security settings or security app like antivirus is blocking the app from being able to access the location to create the user settings file.

This app has always been using the exact same way of saving and loading the user settings since its first release years ago. Were you having the same issue before with version 1.x?
